daemon/algod/api/server/v2/account_test.go

```diff
@@ -102,7 +102,7 @@ func TestAccount(t *testing.T) {
 	b := a.WithUpdatedRewards(proto, 100)
 
 	addr := basics.Address{}.String()
-	conv, err := AccountDataToAccount(addr, &b, map[basics.AssetIndex]string{}, round, a.MicroAlgos)
+	conv, err := AccountDataToAccount(addr, &b, map[basics.AssetIndex]string{}, round, &config.ConsensusParams{MinBalance: 100000}, a.MicroAlgos)
```

Comment:
The minimum balance depends of a multiple fields from `config.ConsensusParams`. It would be wrong to initialize just a single field ( `MinBalance` ), since the function `record.MinBalance` might change over time.
I would suggest you'll grab a specific consensus protocol and use it. i.e. the current protocol sounds like a safe choice here.

Reply:
👍